release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -87,6 +87,42 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-networking"]
 === Networking
 
+[id="ocp-4-13-networking-metrics"]
+==== Enhancements to networking metrics
+[cols="2,2",options="header"]
+.Enhancements to network metrics
+|====
+|Name | Help message
+|`egress_ips_rebalance_total`
+|`The total number of times assigned egress IP(s) were unreachable`
+
+|`egress_ips_node_unreachable_total`
+|`The total number of times assigned egress IP(s) were unreachable`
+
+|`egress_ips_unassign_latency_seconds`
+|`The latency of egress IP unassignment from ovn nb database`
+
+|`interfaces_total`
```

Comment:
This has been renamed: https://github.com/ovn-org/ovn-kubernetes/pull/3391/files